app/workers/terms_of_use/sign_up_service_updater_job.rb

```diff
@@ -5,6 +5,13 @@
 module TermsOfUse
   class SignUpServiceUpdaterJob
     include Sidekiq::Worker
+    sidekiq_options retry: 15 # 2.1 days using exponential backoff
+
+    sidekiq_retries_exhausted do |job, _ex|
+      Rails.logger.warn(
+        "Retries exhausted for #{job['name']} with args #{job['args']}: #{job['error_message']}}"
```

Comment:
Could we prefix this log with the module/class name?

`[TermsOfUse][SignUpServiceUpdaterJob]`

This isn't a platform-wide policy, but I find it easier to query our logs intuitively when we do that

Reply:
Ok, made the update. I was going to set the logger prefix as a constant, but then decided against it since we only have one use of it in the class.